src/com/cli/contact/manager/SearchLastName.java

```diff
@@ -0,0 +1,19 @@
+package com.cli.contact.manager;
+
+import com.cli.contact.model.Person;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SearchLastName implements Searcher {
+    @Override
+    public List<Person> meetSearch(String key, List<Person> persons) {
+        List<Person> meetPersons = new ArrayList<Person>();
+
+        for (Person person : persons) {
+            if(person.getName().equalsIgnoreCase(key) || person.getLastName().equalsIgnoreCase(key) || person.getEmail().equalsIgnoreCase(key)){
```

Comment:
@anghelitalo The name of your class indicates that you will search by last name but you are searching by name and email too, the name of your class must have an accurate description